src/utils.jl

```diff
@@ -186,13 +186,9 @@ end
 Return rotation matrix from vector `u` to vector `v`.
 """
 function uvrotation(u::Vec{2}, v::Vec{2})
-  convert(DCM, CounterClockwiseAngle(∠(u, v)))
+  Angle2d(∠(u,v))
```

Comment:
This can be replaced with `rotation_between` after https://github.com/JuliaGeometry/Rotations.jl/pull/257 is finished.